### DISCUSSION TITLE
Github Copilot auto-deactivated

### DISCUSSION THREAD

**Discussion by loftusa** [community/community#35941] [Copilot Conversations]
Hi, I paid for Github Copilot but it removed my access:
`Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended.`

I just tried cancelling it and re-subscribing, but there's no option anywhere to resubscribe.

How do I get copilot access back? It has been incredibly helpful.

**Comment by davecheney** [comment]
@loftusa i can see that your github username has access to copilot. Do you have more than one github account? If so you may need to sign out of vscode 
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Comment by MaestroJurko** [comment]
I have the same issue and tried this flow, it did not work.

**Reply by ValdemarGr** [reply]
I just encountered this as well, I have a subscription but it seems that the site is also confused.
![image](https://user-images.githubusercontent.com/22582195/202864268-55aae445-2dcb-478c-a6ba-f1bd7ea45d54.png)

Maybe some outage or bug?

**Reply by KibaeKim** [reply]
My subscription ended and I'm trying to restart it. Following the steps [here](https://docs.github.com/en/copilot/quickstart#signing-up-for-github-copilot), on Step 3 I get the following with no option to sign up.

<img width="787" alt="Screen Shot 2022-11-19 at 9 50 04 AM" src="https://user-images.githubusercontent.com/24638190/202864571-1bb8affc-1f08-4655-834f-a0a071b98e3d.png">

**Comment by JoaoChrisostomo** [comment]
I also paid to use github copilot and it's not working in my vsCode I followed every step to use the extension
![image](https://user-images.githubusercontent.com/83593479/202870686-84f98798-6f18-4e41-9dc5-e6489170235b.png)

**Comment by alaabouayed** [comment]
Same for me. I have an active subscription and have had one since August. Today my Copilot stopped working with the following error.

![image](https://user-images.githubusercontent.com/2877958/203867656-c8b674f4-e675-4eb5-99fb-c90e2b2f6b56.png)

I've tried...
- Signing out of GitHub in VSCode and signing back in
- Uninstalling Copilot and reinstalling
- Restarting VSCode
- Restarting my computer

Not sure what else to try.

**Reply by ricardoalmira89** [reply]
i have the same problem, last pay was 3 days ago. i guess this is not a problem in our end. Maybe it's on github...

**Reply by fnyah** [reply]
Same here

**Reply by kevineduardo** [reply]
same here

**Comment by mnj** [comment]
Same issue here in both VS 2022, VC Code, and Rider, signed up for a year, now it's saying its expired not even 1/4 through the year I paid for, with no real way to get it to activate

**Reply by ricardoalmira89** [reply]
i'm on vscode latest version. same problem. 
It was working properly a few minutes ago.

**Comment by GavinRay97** [comment]
Same issue here, I think someone screwed up today with the account activation logic.

I paid 1-year membership on the same day that Copilot went paid. I was a member of the Copilot preview.
As of about a few hours ago, my account stopped working, and the below do not work:
- Uninstalling + Reinstalling the extension (either normal or nightly)
- Signing out of my Github VS Code account, then signing back in

The logs from the extension show the below:

```
[INFO] [auth] [2022-11-24T22:04:00.531Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2022-11-24T22:04:00.532Z] GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."
```

This is fairly frustrating, as $100 is no small bit of change =/

**Reply by cmbaldwin** [reply]
+1 My guess is that it is related to some coding logic applying to users with preview keys still in the settings somewhere? I haven't tried completely re-installing VSCode yet. Has anyone tried that with this more recent issue? (I'd rather not have to do that)

**Comment by nathanclevenger** [comment]
I'm having the exact same issue as well:
```
GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."
```
But I do have an active subscription:
<img width="825" alt="Screenshot 2022-11-24 at 5 31 32 PM" src="https://user-images.githubusercontent.com/4130910/203874417-721f11e1-d112-4c81-84c6-d38f4095c086.png">

**Comment by zhuyuezx** [comment]
I'm having a student account but also same issue here. Seems that it's not a problem on client-side.
![image](https://user-images.githubusercontent.com/56620120/203874822-02807e16-eaba-4071-adde-2843965f9046.png)
![image](https://user-images.githubusercontent.com/56620120/203874857-d804c40e-9616-4dad-93f8-ce31aacba1c2.png)

**Reply by kevineduardo** [reply]
I'm in the same situation.

**Reply by jacksonhuether** [reply]
+1, same issue for me.

**Reply by lcdkhoa** [reply]
the same for me :(

**Comment by foloinfo** [comment]
I'm having the same issue, using nvim with `github/copilot.vim` showing 

```
Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended.

Request Actions:
1. Signup for GitHub Copilot
2. Dismiss
Type number and <Enter> or click with the mouse (q or empty cancels):
```

`:Copilot auth` or `setup` will show that I'm logged in as myself, but `:Copilot status` returns `Copilot: Not authorized`.
Updating the plugin, logout -> login, disable -> enable, restart, nothing works.
I'm subscribed with Yearly plan.

I hope this to be resolved soon.

**Comment by reslear** [comment]
same

**Comment by jamerbi** [comment]
Same issue. I tried in VSCode 2022 and Github Codespaces

**Comment by alanquigley-toast** [comment]
Same

**Comment by sm000k** [comment]
intelij copilot plugin same thing

**Comment by ryan-minato** [comment]
I tried in PyCharm and vscode plugin. Same issue.

**Reply by sm000k** [reply]
true

**Comment by PeterChen1997** [comment]
same

**Comment by fankaiLiu** [comment]
![image](https://user-images.githubusercontent.com/93850567/203883466-504fb35a-0e6f-47d4-92cc-ad1d03a5de23.png)
anger

**Comment by edmar** [comment]
I can confirm the same error.

**Comment by zfben** [comment]
Me too, i tried reinstall the plugin but it doesn't work.

**Comment by sliver37** [comment]
+1 to the list of paid but suddenly not working subscriptions...

How am I expected to code and do my job, without an AI doing it for me!? I'm in too deep to go back now!!

**Comment by Helfull** [comment]
+1
![image](https://user-images.githubusercontent.com/720645/203884999-fe17e7bb-f1e6-4dd2-971e-99c46e27f689.png)

**Comment by dwurf** [comment]
Not working in JetBrains (GoLand) or NVim on Ubuntu.
I can only write code by changing a few characters and pressing Tab so this is a big problem for my work :disappointed:

**Comment by tzv0** [comment]
I see i am not the only one with these issues, this is super unfortunate

**Comment by D0N-B0T** [comment]
same here.

**Comment by HerIsDia** [comment]
Looks like everyone got this problem.

**Comment by emanuelefavero** [comment]
same issue here

**Comment by CalvinRodo** [comment]
same issue

**Comment by ghost** [comment]
same issue +1

**Comment by aidino** [comment]
same issue

**Comment by blackshot** [comment]
same here, i paid for yearly plan... it's frustrating...
![image](https://user-images.githubusercontent.com/3688208/203887970-5b054461-9dce-4b48-a3a5-0935577128f9.png)

**Comment by HiSunzhenliang** [comment]
same. it's frustrating

**Comment by k3yomi** [comment]
Happened to me as well. Glad to hear that I'm not the only one facing the issue. Hopefully it will be restored quite soon.

**Comment by profetia** [comment]
Same issue. Sign out and sign in is of no help

**Comment by yarin-zhang** [comment]
I had the same issue. I tried reinstalling the plugin and login my account again and it didn't solve the problem.

**Comment by 27Bslash6** [comment]
This is where I write a +1 comment right?

**Reply by Royal-lobster** [reply]
Yes ! but we prefer upvoting the post so tread remains free for more detailed discussions

**Comment by i99dev** [comment]
The same issue

**Comment by PhanAnh22021995** [comment]
same issue here.

**Comment by martin-ro** [comment]
same here

**Comment by JackieHanC** [comment]
same here

**Comment by Anonghost720** [comment]
is github going to fix this issue or what

**Comment by theterminalguy** [comment]
same issue

**Comment by Danjar27** [comment]
Did anyone find a way to fix it? It's been a couple of hours since it went down

**Comment by mathminds** [comment]
same issue

**Comment by gaetangr** [comment]
Same issue

**Comment by kevineduardo** [comment]
Same issue here. Student Pack.

**Comment by code-xhyun** [comment]
+1 Same issue

reference link
https://www.githubstatus.com/incidents/69flmdbxhtsk

**Reply by gy0857478** [reply]
Thanks

**Comment by kapiljhajhria** [comment]
same issue paid monthly plan

**Comment by danielravina** [comment]
It's back for me ❤️

**Reply by senecamanu** [reply]
it's back! <3

**Reply by danielravina** [reply]
makes you realize how difficult it is to code without it after using it for a while...

**Reply by senecamanu** [reply]
i know right. i was shocked too when the autocomplete gone away.

**Comment by ricardoalmira89** [comment]
it's working now!

**Comment by zrtsky** [comment]
None of this worked for me, had to completely uninstall vscode with all cache (https://code.visualstudio.com/docs/setup/uninstall#:~:text=Windows%20Installer&text=Search%20for%20Add%20or%20Remove,prompts%20to%20uninstall%20VS%20Code.), I also advise you to create a gist with your settings just in case, sometimes the built-in sync is replaced when reinstalling, here is a good extension for this https://marketplace.visualstudio.com/items?itemName=Shan.code-settings-sync

**Comment by PeterChen1997** [comment]
The problem has been fixed, my vscode back to normal

**Comment by daylonball** [comment]
Working again for me now thankfully

**Comment by reslear** [comment]
not working again 🤬

**Comment by yarin-zhang** [comment]
It looks like there is a new issue with Copilot. 

I recommend subscribe to [Github Status](https://www.githubstatus.com/) to follow up with the recovery progress.

![image](https://user-images.githubusercontent.com/58888890/204066324-84df1776-d1b0-4469-9a01-d6346fbcb1b8.png)

**Comment by jeremiahsherrill** [comment]
> Degraded Performance

If your tires are low on air, that's degraded performance, when the wheels fall of your car, that is something else entirely.
